app/src/main/java/org/mozilla/focus/telemetry/TelemetryWrapper.kt

```diff
@@ -279,3 +284,23 @@ enum class UrlTextInputLocation(internal val extra: String) {
     HOME("home"),
     MENU("menu"),
 }
+
+/** Counts the number of unique home tiles the user has clicked on. */
+private object TelemetryHomeTileUniqueClickCounter {
+
+    @Synchronized // We're doing a get-and-set.
+    fun addTileAndGetCount(context: Context, tile: HomeTile): Int {
+        val sharedPrefs = getSharedPrefs(context)
+        val clickedTileIDs = (sharedPrefs.getStringSet(KEY_CLICKED_HOME_TILE_IDS, null)
+                ?: setOf()).toMutableSet() // create a copy: we can't modify a SharedPref StringSet.
+        clickedTileIDs.add(tile.idToString())
```

Comment:
Should this pref ever be cleared? I'm concerned that over time, this number is basically "only every increasing" (because we never remove anything). "how many unique tiles per session" seems more useful than "unique tiles ever clicked".

Should also cap this at some size, maybe.

Reply:
We decided to move to per-session: I'll update the PR with a new solution.